recipes/server_setup.rb

```diff
@@ -114,37 +121,61 @@
       end
 
       # Create option string
+      force = false
       options = ''
       case volume_values['volume_type']
+      when 'distributed'
+        Chef::Log.warn('You have specified distributed, serious data loss can occur in this mode as files are spread randomly among the bricks')
+        options = ' '
       when 'replicated'
-        # Ensure the trusted pool has the correct number of bricks available
-        if brick_count < volume_values['replica_count']
-          Chef::Log.warn("Correct number of bricks not available: #{brick_count} available, #{volume_values['replica_count']} required for volume #{volume_name}. Skipping...")
+        # Replicated can be anything from a single node to X nodes. Replica_count should equal number of bricks.
+        Chef::Log.warn('You have specified replicated, so the attribute replica_count will be set to be the same number as the bricks you have')
+        node.set['gluster']['server']['volumes'][volume_name]['replica_count'] = brick_count
+        options = "replica #{brick_count}"
```

Comment:
Need to validate `brick_count > 0` otherwise the recipe will execute `gluster volume create chef replica 0` is no peers are defined